pkg/server/batch_check.go

```diff
@@ -88,6 +89,14 @@ func (s *Server) BatchCheck(ctx context.Context, req *openfgav1.BatchCheckReques
 	).Observe(queryCount)
 	span.SetAttributes(attribute.Int("duplicate_checks", metadata.DuplicateCheckCount))
 
+	if metadata.DuplicateCheckCount != 0 {
+		s.logger.InfoWithContext(ctx, "batch check received duplicate checks",
+			zap.String("store_id", storeID),
+			zap.Int("batch size", len(req.GetChecks())),
+			zap.Int("duplicate count", metadata.DuplicateCheckCount),
+		)
+	}
```

Comment:
While this works, its a divergence from how we normally use logs. Lets also consider that this could become noisy for an operator.